### DISCUSSION TITLE
What does best practice manages the airflow dag on the k8s

### DISCUSSION THREAD

**Discussion by allenhaozi** [apache/airflow#27068] [Q&A]
The local tests are fine
 but is there a better way to manage the airflow dag for the k8s cluster and an easy way to maintain the airflow dag

**Comment by potiuk** [comment]
See https://airflow.apache.org/docs/helm-chart/stable/manage-dags-files.html in the official Airlfow Chart (BTW. Using the official chart is recommended if you do not do it yet. It has many things implemented that otherwise you would have to implement in your deployment).

**Reply by allenhaozi** [reply]
thank you @potiuk, Let me try it
